### PR TITLE
(mod/login)link to backend almost done (WIP)

### PR DESCRIPTION
**Pull request by SheItoon**

### CODE REVIEW

File: src/services/Auth.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ class Authentication {
     get() {
         return localStorage.getItem('theedJwt');
     }
-    post(email, password) {
+    post(ev, email, password) {
+        ev.preventDefault(); //don't touch that
```
Comment (djnnvx) [inline src/services/Auth.js:10]: lmao why

**Review by djnnvx — APPROVED** [review]
looks good to me good job

File: src/services/Auth.js (hunk 1 above)
Comment (SheItoon) [reply src/services/Auth.js:10]: Without it the form is not processed client side and the default html form behavior is triggered

Comment (djnnvx) [reply src/services/Auth.js:10]: ok thanks for the clarification

**Pull request merged**